hyperstart.go

```diff
@@ -204,7 +204,7 @@ func (h *hyper) linkPauseBinary(podID string) error {
 
 	pausePath := filepath.Join(pauseDir, pauseBinName)
 
-	return os.Symlink(h.config.PauseBinPath, pausePath)
+	return fileCopy(h.config.PauseBinPath, pausePath)
```

Comment:
The function name should change to reflect the changed behaviour: `copyPauseBinary()` maybe.